src/failures/src/communication/readers/queued-sim-var-reader.test.ts

```diff
@@ -0,0 +1,53 @@
+import { QueuedSimVarReader } from '.';
+import { SimVarReaderWriter } from '..';
+
+test('does not read a value that is not in the collection', async () => {
+    await expectToBeCalledTimes(0, async (r) => {
+        await SimVar.SetSimVarValue(simVarName, 'number', notInCollectionIdentifier);
+        r.update();
+    });
```

Comment:
I'm not a huge fan of being to clever in tests with helper methods, as they tend to obscure the actual call. I find explicit code in tests, even if they are repetitive, more helpful in the case of a test failure (because you're not tracing assertions in sub-functions).

Reply:
I have moved the `expect` calls into the test function body.